vc.adoc

```diff
@@ -1958,6 +1989,388 @@ Content-Type: application/json;charset=UTF-8
+|summary |Object |false |Page summary.
+|summary.limit |Integer |false |Requested size of the page.
+|summary.page |Integer |false |Page number.
+|summary.total |Integer |false |Total count of items.
+|summary.facets |Object |false |Filter values for faceted search.
+|summary.facets.level |Array[Object] |false |Filter values the "level" filter.
+|summary.facets.level[].id |String |false |Filter value the "level" filter.
+|summary.facets.level[].count |Integer |false |Number of search results relevant to the filter value.
+|summary.facets.association |Array[Object] |false |Filter values the "association" filter.
+|summary.facets.instructor |Array[Object] |false |Filter values the "instructor" filter.
+|summary.facets.equipment |Array[Object] |false |Filter values the "equipment" filter.
+|summary.facets.program |Array[Object] |false |Filter values the "category" filter.
+|summary.facets.program[].id |Array[Object] |false |Filter value for the "category" filter.
+|summary.facets.program[].count |Array[Object] |false |Number of search results relevant to the filter value.
+|===
+
+[[_example_request_16]]
+==== link:#_example_request_16[Example request]
+
+[source,highlightjs,highlight]
+----
+$ curl 'http://localhost:8080/api/virtual-classes/v3.0/content-providers/wichita/videos?searchString=yoga&page=0&limit=50' -i -X GET \
+    -H 'Accept: application/json' -H 'authorization: apiKey 0a47c3bf-4740-465d-a22e-00000000ddd0'
+----
+
+[[_example_response_16]]
+==== link:#_example_response_16[Example response]
+
+[source,highlightjs,highlight,nowrap]
+----
+HTTP/1.1 200 OK
+Content-Length: 341
+Content-Type: application/json;charset=UTF-8
+
+{
+    "items": [ {
+        "id": "87",
+        "name": "KIDS YOGA WITH CORRI",
+        "startTime": 1612886400,
+        "endTime": 1612890000,
+        "duration": 3600,
+        "thumbnail": "https://cms.ymca360.org/sites/default/files/live-stream/1/thumb-small.jpg",
+        "instructor": "Corri Lewellen",
+        "level": "BEGINNER",
+        "program": 22,
+        "programName": "Kids' Acitivities",
+        "association": "Wichita"
+    } ],
+    "summary": {
+        "total": 1,
+        "page": 0,
+        "limit": 50,
+        "facets": {
+            "level": [
+                {
+                  "id": "BEGINNER",
+                  "count": 1
+                }
+            ],
+            "association": [ ... ],
+            "program": [ ... ],
+            "instructor": [ ... ],
+            "equipment": [ ... ],
+        }
+    }
+}
+----
+
+
+[[_live_stream_details]]
+== link:#_live_stream_details[Live Stream Details]
+
+[[resources-fetch-live-stream]]
+=== link:#resources-fetch-live-stream[Fetch Live Stream]
+
+`GET /api/virtual-classes/v3.0/content-providers/{provider}/live-stream/{liveStreamId}`
```

Comment:
This line is missing the 's' on live-streams should be:
GET /api/virtual-classes/v3.0/content-providers/{provider}/live-streams/{liveStreamId}

Reply:
That's fixed